Python_Engine/Compute/InstallToolkit.cs

```diff
@@ -41,7 +41,7 @@ public static partial class Compute
         [Input("force", "If you have previously installed a version of Python Toolkit, you may need to force the component to run a new install. This can happen if your system loses files. This is set to false by default, which means if we can detect a previous Python install on your system we will not install Python this time. Set this to true to ignore this check")]
         [MultiOutput(0, "success", "True is Python Toolkit has been successfully installer, false otherwise")]
         [MultiOutput(1, "installedPackages", "A list of the Python packages which have been installed as part of this operation")]
-        public static Output<bool, List<string>> InstallPythonToolkit(bool run = false, bool force = false)
+        public static Output<bool, List<string>> InstallToolkit(bool run = false, bool force = false)
```

Comment:
I have concerns over this because there is no immediate knowledge of what toolkit is being installed. A user might question "could I use this to install XML toolkit?" for example. What are your thoughts @epignatelli ?

Reply:
Ah, sorry - I was relying on the namespaces actuaylly.
This was we have:
- `BH.Engine.Python.Compute.InstallToolkit` and
- `BH.Engine.MachineLearning.Compute.InstallToolkit`

But happy to revert back and change the machine learning instead

Reply:
Hmm I agree the namespace _should_ help, but I fear it might not when we get to the user level on the UIs. You've met users right? 😆

Reply:
Lol, yeah, I don't have a strong opinion on this - moving to the other option